packages/api/tests/marketDirectoryInfo.test.ts

```diff
@@ -79,16 +106,27 @@ const invalidTestCases = [
   ['x', `https://${SOCIAL_DOMAINS['x']}/username!}`], // url with invalid username
 ];
 
-const expectMarketDirectoryInfo = async (
-  farmId: string,
-  expectedData: MarketDirectoryInfoReqBody,
-) => {
-  // @ts-expect-error: TS doesn't see query() through softDelete HOC; safe at runtime
-  const record = await MarketDirectoryInfoModel.query().where({ farm_id: farmId }).first();
```

Comment:
I found it confusing that we were comparing the get response body to db query (same-ish to get response). 

I have changed this so that we are comparing the getResponse to a fake data object we manually inserted with the factories. I thought thats what we normally do -- but maybe I am missing something.